recipes/pysdl2/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "PySDL2" %}
+{% set version = "0.9.4" %}
+{% set md5 = "720b33ed048c169643406ae39ca4dd8d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn:  {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+  patches:
+    - locate_sdl.patch
```

Comment:
Keep the patch here - but propose this patch as a PR on the upstream SDL branch so that other can benefit from it/we have less chance of the patch failing from future upstream changes.